src/lib/db.js

```diff
@@ -379,8 +387,8 @@ export default class db {
       .from('authors')
       .innerJoin('teams_authors', 'authors.id', '=', 'author_id')
       .innerJoin('teams', 'teams.id', '=', 'teams_authors.team_id')
-
       .whereIn('teams.slug', slugs)
+      .orderBy('teams_authors.team_order', 'asc')
```

Comment:
wrong. Should be .author_order.

It will work with just .author_order as we already separate the teams in data object, even though logically it could seem you need to first order by teams and then by authors, but that's actually unnecessary compared to how we process the data afterwards at the moment.